### DISCUSSION TITLE
Output Tab (disable auto open)

### DISCUSSION THREAD

**Discussion by SergioSuarezGil** [dbeaver/dbeaver#11209] [Q&A] [answered]
#### System information: 
- System: MacOS 11.2
-  DBeaver 7.3.4
- Java 8
- MySQL 5.6 DB

It is possible disable auto open output tab? Or move to another tab?

<img width="1155" alt="Captura de pantalla 2021-02-03 a las 12 45 30" src="https://user-images.githubusercontent.com/3636232/106742983-10181d80-661e-11eb-890a-cafd0c25cb91.png">

**Comment by tati-kru** [comment, marked as answer]
You can disable this option in the SQL-editor preferences.
![2021-01-29](https://user-images.githubusercontent.com/30336552/106747753-60e74080-6635-11eb-82fd-66053231e36c.gif)

**Reply by SergioSuarezGil** [reply]
Super, thank you!
